onnxscript/ir/passes/common/unused_removal.py

```diff
@@ -186,3 +186,33 @@ def call(self, model: ir.Model) -> ir.passes.PassResult:
                 modified |= self._process_graph_like(function, used_domains=set())
 
         return ir.passes.PassResult(model, modified=modified)
+
+
+class RemoveUnusedMetadataAndDocStringPass(ir.passes.InPlacePass):
```

Comment:
I don't think there is a notion of "unused" for metadata. For the other passes we always know if something is being "used" in the graph. Suggest renaming

```suggestion
class RemoveMetadataAndDocStringPass(ir.passes.InPlacePass):
```

Reply:
Is there a better file that can host this pass?

Reply:
Created a new file